### DISCUSSION TITLE
Relative paths in Solito

### DISCUSSION THREAD

**Discussion by andreaslydemann** [nandorojo/solito#362] [Q&A]
I'm working on a Solito project that's using Expo Router on native, and just started creating shared routes. I have a profile page on all my tabs (home, messages, settings) and once I navigate to profile and its sub pages I don't want to keep including (and checking which) the tabs every time I navigate. 

In Expo Router relative paths are accepted so you can do push('../profile') and such. As far as I know it's not supported out of the box in Next.js, but the functionality is there on Expo Router at least to utilize (and maybe it could be supported on web by appending router.asPath?)

What are you thoughts? Any plans on supporting relative paths in Solito?

**Comment by nandorojo** [comment]
I don’t exactly have plans to support this honestly, but I suppose feature wise it’s not that hard. I can consider it, though I’ve never found myself wanting it

**Reply by andreaslydemann** [reply]
Got ya 👍 I'm using a simple wrapper around Solito useRouter for now. In any case, my routes are much shorter now. Would be nice to have support in Solito though.
